backend/qemu.pm

```diff
@@ -346,6 +347,59 @@ sub deflate_balloon ($self) {
     $self->handle_qmp_command({execute => 'balloon', arguments => {value => $ram_bytes}}, fatal => 1);
 }
 
+sub save_storage ($self, $args) {
+    my $vars = \%bmwqemu::vars;
+    my $bdc = $self->{proc}->blockdev_conf;
+    my $fname = $args->{filename};
+    my $rsp = $self->handle_qmp_command({execute => 'query-status'}, fatal => 1);
+    bmwqemu::diag("Saving saving storage (Current VM state is $rsp->{return}->{status})");
```

Comment:
This doesn't seem to be covered by unit tests yet

Reply:
yeah, this is why this PR is still in draft mode :D